### PR TITLE
CASSANDRA-16665 May 2021 updates to static pages

### PR DESCRIPTION
**Pull request by ErickRamirezAU**
Updated the following:
- Homepage - replaced "Join World Party" promo with YouTube playlist for party talks recordings
- Ecosystem - added Azure Cosmos DB + edited listing for Ansible collection
- Blog - renamed file for the "Join GSoC 2021" (misspelled "Casssandra" in filename)

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
         <script src="/init.js"></script>
 		<link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400&family=Red+Hat+Display:ital,wght@0,400;0,500;0,700;1,400;1,500&display=swap" rel="stylesheet">
-<script async defer data-domain="cassandra.apache.org" src="https://plausible.cassandra.apache.org/js/plausible.js"></script></head>
+	</head>
```
Comment (michaelsembwever) [inline content/index.html:11]: this line needs to stay. it's provides the tracking we see at https://plausible.cassandra.apache.org/cassandra.apache.org?period=realtime

File: content/ecosystem/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 		<link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400&family=Red+Hat+Display:ital,wght@0,400;0,500;0,700;1,400;1,500&display=swap" rel="stylesheet">
 		<script src="/init.js"></script>
 
-<script async defer data-domain="cassandra.apache.org" src="https://plausible.cassandra.apache.org/js/plausible.js"></script></head>
+	</head>
```
Comment (michaelsembwever) [inline content/ecosystem/index.html:12]: this line needs to stay. it's provides the tracking we see at https://plausible.cassandra.apache.org/cassandra.apache.org?period=realtime

Comment (ErickRamirezAU) [reply content/ecosystem/index.html:12]: I really shouldn't have just accepted what was supplied and instead edit the existing file. Let me redo the PR. Thanks.

File: content/index.html (hunk 1 above)
Comment (ErickRamirezAU) [reply content/index.html:11]: I really shouldn't have just accepted what was supplied and instead edit the existing file. Let me redo the PR. Thanks.

**Comment by ErickRamirezAU** [comment]
Redoing the PR with cleaner updates to existing files instead of replacing them with supplied zip.

File: content/ecosystem/index.html (hunk 2 above)
Comment (michaelsembwever) [reply content/ecosystem/index.html:12]: the plausible tracking is new. i've seen @jtlevy add it to files, but for these custom and temporary html files it is easy to forgot to add this line.

**Pull request closed without merging**